app/models/track.py

```diff
@@ -9,19 +9,21 @@ class Track(db.Model):
     description = db.Column(db.Text)
     track_image_url = db.Column(db.Text)
     color = db.Column(db.String, nullable=False)
+    font_color = db.Column(db.String)
```

Comment:
@niranjan94 ok.. so that means there will be a function in `app/models/track.py`  generating fontcolor from color right?